src/sql/base/browser/ui/infoBox/infoBox.ts

```diff
@@ -160,6 +163,16 @@ export class InfoBox extends Disposable implements IThemable {
 		this._clickListenersDisposableStore.clear();
 	}
 
+	public get ariaLabel(): string {
+		return this._ariaLabel;
+	}
+
+	public set ariaLabel(v: string) {
+		this._ariaLabel = v;
```

Comment:
please rename to make it clear the ariaLabel is for the button not the whole thing.